src/mono/mono/mini/mini-runtime.c

```diff
@@ -1281,6 +1281,8 @@ mono_patch_info_hash (gconstpointer data)
 	}
 	case MONO_PATCH_INFO_GSHAREDVT_IN_WRAPPER:
 		return hash | mono_signature_hash (ji->data.sig);
+	case MONO_PATCH_INFO_X128_GOT:
+		return hash | (guint32)*(double*)ji->data.target;
 	case MONO_PATCH_INFO_R8_GOT:
 		return hash | (guint32)*(double*)ji->data.target;
```

Comment:
This is how the hashes are currently being built for R4/R8, but they don't necessarily make "sense" to me.

This isn't a "good" hash since it just `or's` bits together, it doesn't take into account the "upper bits", and it relies on undefined behavior in the case the `double` or `float` is `NaN`/`Infinity`/`out of range`.

Reply:
It's not the best, but its just used during JITting/AOTing, so it's not a problem in practice.